src/backend/integrations/builtin/obs/effects/toggle-obs-source-filter.ts

```diff
@@ -30,7 +30,29 @@ export const ToggleSourceFilterEffectType: EffectType<EffectProperties> =
           categories: ["common"]
       },
       optionsTemplate: `
-    <eos-container header="Filters">
+    <eos-container ng-show="orphanedSources.length > 0">
+        <div class="effect-info alert alert-warning">
+             <p><b>Warning!</b> There are {{orphanedSources.length}} orphaned source(s) referenced by this effect. 
+             Items in <b>inactive</b> OBS profiles or scene collections <i>will</i> inaccurately show these errors.
```

Comment:
I'm not sure I understand what you mean by this. What are you referring to when saying `these errors`?

Reply:
im not sure what a good name for that would be. we see this only in the error in the log currently and only in the error log. 
what is a good name for that "Items" "Sources" name of thing based on view type?
the intent here is to tell the user that
 Items in inactive OBS profiles or scene collections will inaccurately show these items erroneously and should be ignored until they have changed to that OBS profile or collection.